man/chpl.txt

```diff
@@ -195,6 +195,12 @@ OPTIONS
   --[no-]bounds-checks   Enable [disable] run-time bounds checking,
                     e.g. during slicing and array indexing.
 
+  --[no-]formal-domain-checks   Enable [disable] run-time checks to ensure
```

Comment:
I think there are other sections in here that should be updated. I know --no-checks talks about what checks it turns off